src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/client/PrisonApiClient.kt

```diff
@@ -110,12 +110,12 @@ class PrisonApiClient(
     }
   }
 
-  fun retrieveAgency(agencyId: String): Mono<Agency> {
+  fun retrieveAgency(agencyId: String, activeOnly: Boolean = false): Mono<Agency> {
     val responseType = object : ParameterizedTypeReference<Agency>() {}
     return webClient
       .get()
       .uri { builder ->
-        builder.path("/api/agencies/prison/" + agencyId).build()
+        builder.path("/api/agencies/$agencyId?activeOnly=$activeOnly").build()
```

Comment:
I think you can do `path(...).queryParam("activeOnly", activeOnly)` so you don't need to worry about the syntax of adding a query parameter or if we add more params in the future, etc.